tagstudio/src/qt/helpers/file_deleter.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import pathlib
+import subprocess
+import sys
+import traceback
+from pathlib import Path
+from typing import Callable
```

Comment:
`typing.Callable` is deprecated

```suggestion
from collections.abc import Callable
```